tools/test

```diff
@@ -104,6 +104,7 @@ run_test() {
     local dir=$1
     if [ -z "$NO_GO_GET" ]; then
         go get -t -tags "${TAGS[@]}" "$dir"
+        go mod vendor # re-vendor everything
```

Comment:
Could you explain why we need to run `go mod vendor` in this file?

Reply:
Reason is to avoid error when go get fetches additional depedendency, but doesn’t put it into vendor. In go 1.14, if you have vendor directory, it is used by default, but vendor/modules.txt is verified against go.mod. Without calling go mod vendor here, that verification fails.

Reply:
I would prefer if our test didn’t do additional go get, but I’m not sure why it’s here, so left it for now.
In case of cover tool later in the file, I think that should be part of the build image instead.

Reply:
What if we remove the `-go-get` and `-no-go-get` support at all, always enforcing the behaviour of `NO_GO_GET=true` (current default)? `-go-get` is not used by any script we have, so the only way to use it is to manually call `tools/test -go-get`.

Reply:
> What if we remove the `-go-get` and `-no-go-get` support at all, always enforcing the behaviour of `NO_GO_GET=true` (current default)? `-go-get` is not used by any script we have, so the only way to use it is to manually call `tools/test -go-get`.

I will remove it. This doesn't fix problem with `cover` tool, used when run under CIRCLECI.